frontend/actions/index.js

```diff
@@ -93,6 +93,8 @@ export const fetchRechargeCart = () => (dispatch, getState) => {
     .catch((err) => {
       logger.error(err);
       dispatch(errorRechargeCart());
+      // TODO: show modal and redirect user to homepage
```

Comment:
@aaron-martin do you think we should redirect the user to the homepage here? 
So he has to enter cart etc again and trigger the requests automatically

Reply:
@Carsten89 I think that is a good idea.